src/components/LearnNavBar/LearnNavBar.tsx

```diff
@@ -1,14 +1,19 @@
 import { LearnNavBarDesktop } from "./LearnNavBarDesktop";
```

Comment:
Is this LearnNavBar being used? can it just be removed completely?

Reply:
Can be removed, thanks for calling it out. I definitely need to clean it up some more